database.py

```diff
@@ -16,6 +16,17 @@ class Roles(set):
     pass
 
 
+class StudentCourse:
+    def __init__(self, discord_id, semester, lva_nr, active):
```

Comment:
@xMuffin please use type hints for the parameters to improve readability and to make possible future changes easier to understand